hepcrawl/spiders/desy_spider.py

```diff
@@ -121,62 +177,25 @@ def crawl_local_directory(self):
                 callback=self.parse,
             )
 
-    def crawl_ftp_directory(self):
-        ftp_host, ftp_params = ftp_connection_info(
-            self.ftp_host,
-            self.ftp_netrc,
-        )
-
-        remote_files_paths = ftp_list_files(
-            self.ftp_folder,
-            destination_folder=self.destination_folder,
-            ftp_host=ftp_host,
-            user=ftp_params['ftp_user'],
-            password=ftp_params['ftp_password'],
-            only_missing_files=False,
-        )
-
-        xml_remote_files_paths = self._filter_xml_files(remote_files_paths)
-
-        for remote_file in xml_remote_files_paths:
-            self.logger.info(
-                'Remote: Try to crawl file from FTP: {0}'.format(remote_file),
-            )
-            remote_file = str(remote_file)
-            ftp_params['ftp_local_filename'] = os.path.join(
-                self.destination_folder,
-                os.path.basename(remote_file),
-            )
-            remote_url = 'ftp://{0}/{1}'.format(ftp_host, remote_file)
-            yield Request(
-                str(remote_url),
-                meta=ftp_params,
-                callback=self.handle_package_ftp,
-            )
-
-    def handle_package_ftp(self, response):
-        """Yield every XML file found.
-
-        This is an intermediate step before calling :func:`DesySpider.parse`
-        to handle ftp downloaded "record collections".
-
-        Args:
-            response(hepcrawl.http.response.Response): response containing the
-                information about the ftp file download.
-        """
-        self.logger.info('Visited url {}'.format(response.url))
-        file_path = response.body
-        yield Request(
-            'file://{0}'.format(file_path),
-            meta={'source_folder': file_path},
-            callback=self.parse,
-        )
+    def crawl_s3_bucket(self):
+        input_bucket = self.s3_resource.Bucket(self.s3_input_bucket)
+        existing_files = os.listdir(self.destination_folder)
```

Comment:
I would forget about the local files handling, it's not needed if you move the source files to a different bucket.

Reply:
I would prefer to leave `local` part of spider intact, as it's not part of this task.